app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split()
+    if len(split_command) == 3:
+        parameter, old_file, new_file = split_command
+
+        if parameter != "cp":
+            raise ValueError("Parameter must be 'cp'")
+
+        if old_file == new_file:
+            raise ValueError("old_file and new_file must have different names")
```

Comment:
It must do nothing
U don't need to raise any errors
https://github.com/mate-academy/py-copy-file#:~:text=It%20must%20do%20nothing%20in%20case%20the%20user%20is%20trying%20to%20copy%20file%20to%20file%20with%20the%20same%20name.
![image](https://user-images.githubusercontent.com/86322722/229278366-35135080-6c19-4012-886b-abb46ea084b1.png)